### PR TITLE
Adds support for getting single properties

### PR DESCRIPTION
**Pull request by sammyd**
The HS API allows for requesting of single properties and groups, but this functionality has yet to be implemented in ruby. This PR adds this functionality for the currently supported properties: contact, company and deal.

I think I've added tests correctly, but please let me know if not. I might also have overlooked the reasoning as to why this isn't already implemented, or maybe I missed where it /is/ implemented. I'd appreciate you taking a look and consider merging it for the released gem if appropriate.

Thanks!

### CODE REVIEW

File: lib/hubspot/company_properties.rb [hunk 1]
```diff
@@ -19,10 +21,18 @@ def all(opts={}, filter={})
         superclass.all(ALL_PROPERTIES_PATH, opts, filter)
       end
 
+      def get(property_name, opts={})
```
Comment (fonji) [inline lib/hubspot/company_properties.rb:24]: shouldn't it be `find` ?
```suggestion
      def find(property_name, opts={})
```

File: lib/hubspot/company_properties.rb [hunk 2]
```diff
@@ -19,10 +21,18 @@ def all(opts={}, filter={})
...
+        superclass.get(GET_PROPERTY_PATH, property_name, opts)
+      end
+
       def groups(opts={}, filter={})
         superclass.groups(ALL_GROUPS_PATH, opts, filter)
       end
 
+      def get_group(group_name, opts={})
```
Comment (fonji) [inline lib/hubspot/company_properties.rb:32]:
```suggestion
      def find_group(group_name, opts={})
```

**Review by fonji — COMMENTED** [review]
Looks good to me! I'm just wondering if `get` should be `find`.
@Intrepidd do you have an opinion?

Sorry for the delay, I changed my notifications to be faster next time.

**Comment by sammyd** [comment]
I think my original logic was that properties are indexed by name rather than `id`, but that doesn't make a whole lot of sense. I'll update it to `find` and resubmit. Thanks!

**Comment by fonji** [comment]
Don't worry @sammyd, once more our bad: PR from forks don't have access to secrets.
We're learning

**Comment by sammyd** [comment]
OK! Let me know if you need me to to anything to try and work around it.

**Comment by fonji** [comment]
[this documentation seems interesting](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)
It's a guess but I think you should set up the secrets on your side and check that box
image

**Comment by sammyd** [comment]
Happy to check that box, but I can't find it on this PR. I can add those secrets to this repo—do they need values that will allow them to access the HS API, or do they just need to be present?

**Comment by sammyd** [comment]
I have tried adding sample (non-valid) secrets to my fork of the repo, and have pushed a tiny commit to try and retrigger the action—I didn't realise that you would need to approve the workflow from running. Let me know if this has any success.

**Comment by fonji** [comment]
Bummer, it didn't work.
We will have to understand this sooner or later, but I want this shipped, so I'll create a branch here for now.
I hope I'll get it before next PR.

**Comment by fonji** [comment]
v.0.11.0 released

**Pull request merged**